inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -518,35 +518,8 @@
             },
             "type": "array"
         },
-        "subject_terms": {
-            "uniqueItems": true,
-            "items": {
-                "type": "object",
-                "properties": {
-                    "source": {
-                        "enum": [
-                            "automatically added based on DCC, PPF, DK",
-                            "submitter",
-                            "INSPIRE"
-                        ],
-                        "type": "string"
-                    },
-                    "term": {
-                        "type": "string"
-                    },
-                    "scheme": {
-                        "enum": [
-                            "INSPIRE",
-                            "arXiv",
-                            "APS",
-                            "PoS",
-                            "Elsevier"
-                        ],
-                        "type": "string"
-                    }
-                }
-            },
-            "type": "array"
+        "field_categories": {
+            "$ref": "elements/fields.json"
```

Comment:
I think it's better if elements define singular stuff (a field, instead of fields) so that they are more flexible. One can always write
```JSON
{
    "field_categories": {
        "type": "array",
        "items": {"$ref": "elements/field.json"},
        "uniqueItems": true
    }
}
```
Since this is how it was originally, what was the reason to change it to plural? @tomaszgy @bittirousku @kaplun

(Same thing applies to titles)